tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,31 @@
+""" Handles logging functionality. """
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+
+def log(log_priority,publisher_id,**log_payload):
+    """
+    Function used for logging errors/output/info.
+    Args:
+        log_priority : Priority of the Log Message (error/info)
+        publisher_id : Id of publisher (mandatory)
+        log_payload  : Payload can contain parameters like message that is to be logged
+    """
+    try:
+        if publisher_id != None:
+            Event(
+                Message(
+                    priority=log_priority,
+                    publisher=publisher_id,
+                    payload=log_payload
```

Comment:
Other than the given fields above(priority, publisher, payload) ,more fields are to be taken care of for job updates. You can find them here https://github.com/Tendrl/commons/blob/develop/tendrl/commons/message.py#L33